samples/nrf5340/netboot/Kconfig

```diff
@@ -8,6 +8,19 @@ menu "nRF5340 Network Core Bootloader Sample Settings"
 
 config NETBOOT_MIN_PARTITION_SIZE
 	bool "Use minimimum partition size"
+
+config B0N_SIZE
+	hex
+	default 0x8580 if PM_PARTITION_SIZE_PROVISION=0x280
+	default 0x2580 if PM_PARTITION_SIZE_PROVISION=0x280 && NETBOOT_MIN_PARTITION_SIZE
+	default 0x8800
```

Comment:
This is very problematic for moving to sysbuild in ncs, where images are not generated using child/parent images, this information needs to be known by all images which would have been possible before using PM but moving this to a single application's Kconfig is going to remove that ability, therefore marking as NACK until we can assess the scope of what this entails